tests/integration/security/authorization_test.go

```diff
@@ -361,22 +355,22 @@ func TestAuthorization_Deny(t *testing.T) {
 						}
 					}
 					cases := []rbacUtil.TestCase{
-						newTestCase(b, "/deny", false),
-						newTestCase(b, "/deny?param=value", false),
-						newTestCase(b, "/global-deny", false),
-						newTestCase(b, "/global-deny?param=value", false),
-						newTestCase(b, "/other", true),
-						newTestCase(b, "/other?param=value", true),
-						newTestCase(b, "/allow", true),
-						newTestCase(b, "/allow?param=value", true),
-						newTestCase(c, "/allow/admin", false),
-						newTestCase(c, "/allow/admin?param=value", false),
-						newTestCase(c, "/global-deny", false),
-						newTestCase(c, "/global-deny?param=value", false),
-						newTestCase(c, "/other", false),
-						newTestCase(c, "/other?param=value", false),
-						newTestCase(c, "/allow", true),
-						newTestCase(c, "/allow?param=value", true),
+						newTestCase(dst0, "/deny", false),
```

Comment:
I think this still reduce the current test coverage:

In the old test, we tested all the 16 cases on k8s workload (even they are different but it's both k8s).

But in the new test, we only test the first 8 cases on k8s workload and then the next 8 cases on the vm workload. This means the first 8 cases are not tested on vm workload, and the last 8 cases are not tested on k8s workload.

Could we make the test to run all cases on both platform:
- Test k8s workload b and c (current case)
- Test vm workload vm1 and vm2 (new test, just replace b with vm1 and c with vm2)

Reply:
I have reverted back for b and c, the policies are still templatized.
I also added more policies for vm, the policy is similar to c. This is not perfect, but I think it is good as an initial step. Now we have test policy templatized, and vm included. How to revise the policies for better coverage for vm could be left for future work. What do you think?

Reply:
Sounds good, we can always add more tests later but we should not reduce the test coverage of current tests.